Update README.md

AndreiCherniaev opened the pull request.

AndreiCherniaev committed:
Update README.md

Thanks to https://www.reddit.com/r/civ/comments/58tepp/where_is_the_save_location_for_civ_6/

iqqmuT commented:
Thank you @AndreiCherniaev ! I suggest to have this format:

```
* Linux: `~/.local/share/aspyr-media/Sid Meier's Civilization VI/Saves/`
* Windows: `%USERPROFILE%\Documents\My Games\Sid Meier's Civilization VI\Saves`
```

Steam and Epic seem to have a little bit different folder on Windows but user should get the idea

AndreiCherniaev committed:
Update README.md

Thanks to https://www.reddit.com/r/civ/comments/58tepp/where_is_the_save_location_for_civ_6/

AndreiCherniaev commented:
Let's use steam format for Linux: `Sid Meier's Civilization VI/Saves/` and Epic format for Windows: `Sid Meier's Civilization VI (Epic)\Saves`. We can reduce a lot of boring text like "if you use STEAM version, if you use EPIC version" but also became clue for users...

AndreiCherniaev commented:
Thanks for approving my commit! 
By the way, can you approve my friend request on linkedin please?
